.github/workflows/build-publish.yml

```diff
@@ -19,20 +19,14 @@ jobs:
           submodules: 'recursive'
 
       - name: Set up JDK 11
-        uses: actions/setup-java@v2
+        uses: actions/setup-java@v3
         with:
           distribution: 'zulu'
           java-version: '11'
+          cache: 'gradle'
```

Comment:
I'm not sure if we need caching here because the gradle-build-action takes care of caching now